### PR TITLE
Add .python-version file during provisioning

### PR DESCRIPTION
**Pull request by marknach**
part of #167, added in a .python-version file during provisioning step

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -12,6 +12,7 @@ node_modules
 # local env files
 .env.local
 .env.*.local
+.python-version
```
Comment (marknach) [inline .gitignore:15]: up for feedback if you want to just commit this file vs adding it in during the vagrant provisioning

File: vagrant_scripts/provision.sh [hunk 2]
```diff
@@ -48,13 +48,14 @@ eval "$(pyenv virtualenv-init -)"
 pyenv install 3.6.8
 pyenv virtualenv 3.6.8 swarfarm-3.6.8
 pyenv activate swarfarm-3.6.8
+cd /vagrant
+pyenv local swarfarm-3.6.8
```
Comment (marknach) [inline vagrant_scripts/provision.sh:51]: assuming we want this local to the /vagrant directory, but let me know if you think different

File: .gitignore (hunk 1 above)
Comment (PeteAndersen) [reply .gitignore:15]: I'm on board with how you've handled. It's more related to how we've chosen to set up the python environment, and not really part of the project source itself, so this is good.

File: vagrant_scripts/provision.sh (hunk 2 above)
Comment (PeteAndersen) [reply vagrant_scripts/provision.sh:51]: I believe it's required to be local to the /vagrant dir

**Review by PeteAndersen — DISMISSED** [review]

File: vagrant_scripts/provision.sh [hunk 3]
```diff
@@ -48,13 +48,14 @@ eval "$(pyenv virtualenv-init -)"
 pyenv install 3.6.8
 pyenv virtualenv 3.6.8 swarfarm-3.6.8
 pyenv activate swarfarm-3.6.8
```
Comment (PeteAndersen) [inline vagrant_scripts/provision.sh]: This activate line is technically no longer required since the next line activates the venv as part of the command

**Review by PeteAndersen — CHANGES_REQUESTED** [review]

File: vagrant_scripts/provision.sh (hunk 3 above)
Comment (PeteAndersen) [reply vagrant_scripts/provision.sh]: The `pyenv deactivate` command further down should also be removed

Comment (marknach) [reply vagrant_scripts/provision.sh]: good catch, updated

**Comment by marknach** [comment]
retested this after latest changes via removing the .python-version file and running `vagrant provision`

**Review by PeteAndersen — APPROVED** [review]

**Pull request merged**